packages/ckeditor5-indent/tests/indentblockcommand.js

```diff
@@ -132,6 +132,15 @@ describe( 'IndentBlockCommand', () => {
 					setData( model, '<paragraph blockIndent="indent-4">f[]oo</paragraph>' );
 					expect( command.isEnabled ).to.be.false;
 				} );
+
+				// Should be disabled for block items in Document Lists. See https://github.com/ckeditor/ckeditor5/issues/14155.
+				it( 'should be false for a block element inside a list item', () => {
```

Comment:
Please add tests for outdent to confirm that it is possible for the document list.